llmfoundry/models/hf/hf_fsdp.py

```diff
@@ -94,7 +94,12 @@ def hf_get_hidden_layers(model: PreTrainedModel):
         'model.layers',  # LLaMa
         'transformer.blocks',  # MPT
     )
-    return findattr(model, hidden_layers_attrs)
+    layers = findattr(model, hidden_layers_attrs)
```

Comment:
Functional change: this method will now raise an error if no hidden layers found.

Seems safe given the all it's current invocations make this assumption anyway, but please vet.

Reply:
yeah, seems good to me.